modules/nw-ne-comparing-ingress-route.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-comparing-ingress-route_{context}"]
+= Comparing Routes and ingresses
+The most common way to manage ingress traffic is with the Ingress Controller. OpenShift implements the Ingress Controller with a shared router service that runs as a pod inside the cluster. You can scale and replicate this pod like any other regular pod. This router service is based on link:http://www.haproxy.org/[HAProxy], which is an open source load balancer solution.
+
+Ingress traffic accesses services in the cluster through a route. Routes and ingresses are the main resources for handling ingress traffic. Ingress is a Kubernetes resource that provides features similar to a route. Ingress accepts external requests and delegates them based on the route. You can only allow certain types of connections: HTTP/2, HTTPS and server name identification (SNI), and TLS with certificate. In OpenShift, routes are generated to meet the conditions specified by the ingress object.
```

Comment:
Paragraphs 2 and 3 might be improved by making more explicit which resource you're talking about. The trouble is, both the ingress and route methods deal with ... ingress and routes, so it's a bit hard to tell the difference. 

Consider, for example, starting each paragraph with something along the lines of this:

1. The Kubernetes `Ingress` resource allows you to do this ...
2. The OpenShift `Route` resource allows you to do that ... 

That way, even if the text is a mix of ingress/route discussion, at least we've anchored a clear label on the reader's mind.